calico-cloud/get-started/system-requirements.mdx

```diff
@@ -0,0 +1,47 @@
+---
+description: Review cluster requirements to connect to Calico Cloud.
+---
+
+# System requirements
+
+Before you connect your cluster to Calico Cloud, make sure your cluster meets the system requirements.
+Your cluster must already have a CNI installed before you can connect to Calico Cloud.
+
+<!--  This file is referenced in the manager UI and its location should not be changed with accounting for references from pendo/manager UI. A symlink so this file exists at install/system-requirements.  -->
+
+## Kubernetes distributions and CNIs
+
+Calico Cloud works with Kubernetes on self-provisioned infrastructure and on managed Kubernetes distributions.
+To use Calico Cloud for both networking and network policy, your cluster must have Calico Open Source installed before you connect to Calico Cloud.
+For most managed distributions, you can use the provider's CNI for networking and use Calico Cloud for network policy.
+
+| Distribution | CNI |
+| --- | --- |
+| Kubernetes on self-provisioned infrastructure | - Calico Open Source 3.26 and 3.27 |
+| Amazon Elastic Kubernetes Service | - Calico Open Source 3.26 and 3.27<br/> - Amazon VPC CNI |
+| Azure Kubernetes Service | - Calico Open Source 3.26 and 3.27<br/> - Azure CNI |
+| Google Kubernetes Engine | - Calico Open Source 3.26 and 3.27<br/> - GKE CNI |
+| Rancher Kubernetes Engine 2 | - Calico Open Source 3.26 and 3.27 |
+
+
+## Kubernetes versions
+
+* Your Kubernetes distribution must be based on Kubernetes 1.25 to 1.29
+
+## Architectures
+
+* Calico Cloud can be installed on nodes based on x86-64 and ARM64 architectures.
+  :::note
+  For the GKE managed Kubernetes service, Calico Cloud supports only the x86-64 architecture.
+  :::
```

Comment:
This is no longer needed, we've resolved the issue with GKE ARM clusters now

Reply:
Thanks, I've removed the note.